src/templates/commander.html

```diff
@@ -1,4 +1,4 @@
 <div class="fd-commander">
-    <input type="text" placeholder="<%-gettext('Enter command')%>" />
-    <button class="fd-commander-close"><i class="fa fa-close"></i></button>
+    <i class="fa fa-close fd-commander-close"></i>
+    <input type="text" class="form-control" placeholder="<%-gettext('Enter command')%>" />
```

Comment:
I find this to be an annoying CSS/HTML interaction. The close button comes after the input in the final layout, and it should be the same in the HTML! (ranting at HTML/CSS, not @orangejenny)